mycore-neo4j/src/main/java/org/mycore/mcr/neo4j/index/MCRNeo4JIndexEventHandler.java

```diff
@@ -53,46 +53,50 @@ public class MCRNeo4JIndexEventHandler extends MCREventHandlerBase {
     private static final long DELAY_IN_MS = MCRConfiguration2
         .getLong(MCRNeo4JConstants.NEO4J_CONFIG_PREFIX + "DelayIndexing_inMS").orElse(2000L);
 
+    private static final boolean ENABLED = MCRConfiguration2
+        .getString(MCRNeo4JConstants.DEFAULT_NEO4J_SERVER_URL).isPresent();
+
     private static final DelayQueue<MCRDelayedRunnable> NEO4J_TASK_QUEUE = new DelayQueue<>();
 
     private static final ScheduledExecutorService NEO4J_TASK_EXECUTOR = Executors.newSingleThreadScheduledExecutor();
 
     static {
+        if (!ENABLED) {
+            LOGGER.info("Neo4J Indexing is disabled");
```

Comment:
maybe add: because the property MCRNeo4JConstants.DEFAULT_NEO4J_SERVER_URL ist not set